lib/rules/web/admin_console/4.5/deployment.xyaml

```diff
@@ -96,4 +96,15 @@ check_current_cpu_utilization:
   element:
     selector:
       <<: *value
+check_pod_status:
+  elements:
+  - selector: &status
+      xpath: //tr/td[contains(.,'<pod_name>')]/following-sibling::td[2][contains(.,'<pod_status>')]
+    timeout: 120
```

Comment:
Do we need so long timeout here?

Reply:
Yeah. We need to wait the pod changed from containerCreating to ImagePullBackOff  on console. https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/84416/artifact/embedded_files/2020-04-13T06%3A15%3A19+00%3A00-screenshot.png